assets/src/dashboard/components/cardGrid/index.js

```diff
@@ -55,9 +52,8 @@ CardGrid.propTypes = {
 };
 
 export const StoryGrid = styled(CardGrid)`
-  grid-column-gap: 24px;
-  grid-row-gap: 24px;
   margin: 20px;
+  width: calc(100% - 40px);
```

Comment:
max width minus left and right margin to get the grid to have the right available width to spread over